packages/SwingSet/tools/run-utils.js

```diff
@@ -103,11 +105,29 @@ export const makeRunUtils = (controller, harness) => {
   // promise that can remain pending indefinitely, possibly to be settled by a
   // future message delivery.
 
+  /**
+   * `freeze` but not `harden` the proxy target so it remains trapping.
+   * Although a frozen-only object will not be defensive, since it could still
+   * be made non-trapping, these are encapsulated only within proxies that
+   * will refuse to be made non-trapping, and so can safely be shared.
+   * @see https://github.com/endojs/endo/blob/master/packages/ses/docs/preparing-for-stabilize.md
+   */
+  const objTarget = freeze({});
+
+  /**
+   * `freeze` but not `harden` the proxy target so it remains trapping.
+   * Although a frozen-only object will not be defensive, since it could still
+   * be made non-trapping, these are encapsulated only within proxies that
+   * will refuse to be made non-trapping, and so can safely be shared.
+   * @see https://github.com/endojs/endo/blob/master/packages/ses/docs/preparing-for-stabilize.md
+   */
+  const funcTarget = freeze(() => {});
+
   /** @type {EVProxy} */
   // @ts-expect-error cast, approximate
   const EV = Object.assign(
     presence =>
-      new Proxy(harden({}), {
+      new Proxy(funcTarget, {
```

Comment:
Why use `funcTarget` instead of `objTarget` here and below?  `EV` does not support remotable functions at this time, and has various other deficiencies when compared with `E`.  I'd rather fix those deficiencies in a separate PR instead of just making partial changes that aren't fully baked.

Reply:
This comment still applies.  Do you have an opinion, @erights?

Reply:
Yes, I will do that.

Reply:
Done